packages/google_sign_in/google_sign_in_platform_interface/lib/src/types.dart

```diff
@@ -4,15 +4,37 @@
 
 import 'package:quiver/core.dart';
 
-enum SignInOption { standard, games }
+/// Holds default configuration options to use when signing in.
+enum SignInOption {
```

Comment:
Should these options link to https://developers.google.com/android/reference/com/google/android/gms/auth/api/signin/GoogleSignInOptions.html#public-static-final-googlesigninoptions-default_games_sign_in
and 
https://developers.google.com/android/reference/com/google/android/gms/auth/api/signin/GoogleSignInOptions.html#DEFAULT_GAMES_SIGN_IN

Obviously, developers that use this plugin should already know what they need, so it really is just whether we could link to it. I'm not familiar with Google Sign In and would then wonder what happened to the user ID once logged in with the Games Sign In.

Reply:
Done.